Marlin/Configuration.h

```diff
@@ -2174,11 +2190,13 @@
 //
 // ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
 //
-//#define TOUCH_BUTTONS
-#if ENABLED(TOUCH_BUTTONS)
+//#define TOUCH_SCREEN
+#if ENABLED(TOUCH_SCREEN)
```

Comment:
If `TOUCH_BUTTONS` is going away completely as a separate user-configurable option, then I will follow up shortly with a suitable conditional to replace it. There are several pins files that still refer to `TOUCH_BUTTONS` and I want make it clear this refers specifically to the `XPT2046` device.

Reply:
Will the xpt2046 of the emulated DOGM migrated to the new implementation? Right now, I think every TOUCH_BUTTONS pin refer to specifically emulated DOGM and software spi.